kitsune/products/jinja2/products/includes/topic_macros.html

```diff
@@ -9,14 +9,12 @@
       </a>
     </li>
     {% if loop.last %}
-      {% if show_community_support %}
```

Comment:
This variable is no unused. It should be removed from the macro, and any places this macro is called.